core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -104,11 +93,9 @@ public Configuration(Properties props) {
 
   /**
    * Default constructor.
-   *
-   * Most clients will call this constructor to allow default loading of properties to happen.
    */
   public Configuration() {
```

Comment:
I removed the references to this `public constructor` in production code in this PR.
for test code, there are > 60 references, I think it might be better to have separate PR doing that.